product_docs/docs/biganimal/release/getting_started/02_connect_cloud_account.mdx

```diff
@@ -2,39 +2,37 @@
 title: "Connecting your BigAnimal account"
 ---
 
-This topic describes how to set up your BigAnimal account on Azure Marketplace. Your Azure subscription for BigAnimal is where you create and manage Postgres clusters.
+Set up your BigAnimal account on Azure Marketplace. Your Azure subscription for BigAnimal is where you create and manage Postgres clusters.
```

Comment:
Based on a discussion with Ben and Dee Dee we decided to replace all references to clusters (such as Postgres clusters, EDB clusters, and so on) with the term "clusters".

Reply:
Here is a link to that conversation: https://github.com/EnterpriseDB/docs/pull/1855#discussion_r731114575